library/allow_cronjobs.php

```diff
@@ -0,0 +1,23 @@
+<?php 

```

Comment:
this needs a author/license/copyright header. see here for example:
http://www.open-emr.org/wiki/index.php/How_to_Document_Your_Code_Properly#File-Level_DocBlock